_health-care/_js/_form.js

```diff
@@ -1,21 +1,47 @@
 var HealthApp = (function() {
   "use strict";
 
-  var identityTransform = function(formData, arg) { return arg; };
-  var extractTransform = function(formData, arg) { return formData[arg]; };
+  function identityTransform(formData, arg) { return arg; }
+  function extractTransform(formData, arg) { return formData[arg]; }
```

Comment:
that's my reading of it...

This is all cause ES5 and earlier don't have any well defined module system so people are hacking in features using functions. It's ironic that JS, the butt of language jokes, is using all the techniques developed by LISP, the common ivory tower of language purists. :-/

Anyways, ES6 seems to have solved this by making it so top-level functions aren't in global (window?) scope by default which makes them way more useful.

http://www.sitepoint.com/understanding-es6-modules/